doc/api/unicode.md

```diff
@@ -0,0 +1,55 @@
+# icu
+
+    Stability: 1 - Experimental
+
+The `unicode` module provides an interface to Unicode and Internationalization
+functionality provided by the ICU4C library.
+
+```js
+const unicode = require('unicode');
+```
+
+## unicode.transcode(buf, from_enc, to_enc)
+
+* `buf` {Buffer} A `Buffer` instance
+* `from_enc` {string} The current encoding
+* `to_enc` {string} The target encoding
+
+Re-encodes the given `Buffer` from one character encoding to another. Returns
+a new `Buffer` instance.
```

Comment:
It may be less surprising if it was always returning a new Buffer. Like `Buffer.concat` when the list has only one Buffer.